examples/seismic/abc_methods/04_habc.ipynb

```diff
@@ -0,0 +1,1593 @@
+{
```

Comment:
<p>I am afraid I am not able to follow the equations here. Maybe some more senior should have a look.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
OK!